<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ZCBLRLP0293" failures="1" tests="3" name="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Invalid_TestCases" time="85.945" errors="0" timestamp="2024-07-06T15:43:26 IST" skipped="0">
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Invalid_TestCases" name="verifyTransactionAmountShouldNotBeEmpty" time="8.351"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Invalid_TestCases" name="emptyTxnAmountButGoToPassPointValue" time="8.564"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Invalid_TestCases" name="goTOPassPointMoreThanTransactionAmount" time="69.030">
    <failure message="An error occurred while verifying the Validation message: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator &#039;By.xpath: //p[@id=&#039;EP041-helper-text&#039; and contains(text(),&#039;Entered Go To Pass points used is more than the Transaction Amount&#039;)]&#039; (tried for 10 second(s) with 500 milliseconds interval)" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: An error occurred while verifying the Validation message: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //p[@id='EP041-helper-text' and contains(text(),'Entered Go To Pass points used is more than the Transaction Amount')]' (tried for 10 second(s) with 500 milliseconds interval)
at org.testng.Assert.fail(Assert.java:111)
at zapcg.Capillary.PageObject.earnPointsPage.verifyValidationMessageIfGoToPassIsGreaterThanTxnAmt(earnPointsPage.java:839)
at zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Invalid_TestCases.goTOPassPointMoreThanTransactionAmount(EarnPoint_Invalid_TestCases.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- goTOPassPointMoreThanTransactionAmount -->
</testsuite> <!-- zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Invalid_TestCases -->
